src/Internal.AspNetCore.SiteExtension.Sdk/build/Internal.AspNetCore.SiteExtension.Sdk.targets

```diff
@@ -23,7 +23,6 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Remove="Internal.AspNetCore.Sdk" />
-    <PackageReference Remove="Microsoft.NETCore.App" />
```

Comment:
Something changed in SDK and we don't need this anymore

Reply:
Probably related to `FrameworkReference` changes.